README.md

```diff
@@ -1 +1,39 @@
-# code-challenges-javascript
\ No newline at end of file
+# code-challenges-javascript
+This repo is for Javascript code-challenges
+## Set up
+1) cd into the directory you want this repository in - clone it down
+```
+git clone <path>
+```
+2) install jest for testing
+```
```

Comment:
This is incorrect.  `jest` is already listed as a development dependency in your [package.json ](https://github.com/aepaysinger/code-challenges-javascript/blob/main/package.json#L20)

JavaScript how to install dev dependencies package.json.